src/doc/man/includes/options-target-triple.md

```diff
@@ -1,6 +1,7 @@
 {{#option "`--target` _triple_"}}
-{{actionverb}} for the given architecture. The default is the host
-architecture. The general format of the triple is
+{{actionverb}} for the given architecture.{{#if target-default-to-all-arch}}
+The default is all architectures.{{else}} The default is the host architecture.
+{{/if}} The general format of the triple is
```

Comment:
![image](https://user-images.githubusercontent.com/14314532/154532971-0130372c-1ce1-4224-894f-678d50a5ba4f.png)


NIT: some whitespace control might be better? For example,

```suggestion
{{actionverb}} for the given architecture.
{{~#if target-default-to-all-arch}} The default is all architectures.
{{~else}} The default is the host architecture.
{{~/if}} The general format of the triple is
```